backend/server.js

```diff
@@ -0,0 +1,126 @@
+const express = require('express')
+
+
+const postgresql= require(  "./postgresql");
+
+
+const app = express();
+
+let bodyParser = require('body-parser');
+const port = 3000
+
+app.use(express.json());
+app.use(express.urlencoded({ extended: true }));
+app.use(bodyParser.json());
+app.use(bodyParser.urlencoded({extended: true}));
+
+
+
+
+app.get("/check/:poll_name", async (req, res) => {
+   
+
+
+    const rows = await postgresql().query(
+        `select exists(select 1 from polls where pollName='${req.query.poll_name}')`
```

Comment:
Should `req.params` instead of `req.query`

Reply:
No its working this way